.flowconfig

```diff
@@ -1,5 +1,5 @@
 [ignore]
-<PROJECT_ROOT>/node_modules/.*
+node_modules/resolve/test/*
```

Comment:
There's a random malformed `package.json` in this directory that Flow complains about. 🤷‍♂️